Note about Override in Outlook in the Web (#11458)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/use-notifications-and-policy-tips.md

- If NotifyAllowOverride action is set with WithoutJustification or WithJustification or FalsePositives, make sure BlockAccess is set to true and BlockAccessScope has appropriate value. Otherwise policy tip will come up but the user will not find an option to override the email with justification.

> [!NOTE]

> >  In Outlook in the Web to be able to see Override in Policy Tip, the DLP policy needs to be "Turn On", since in Test Mode (with notifications) it won´t be display. As well as, in DLP configuration some restriction action needs to be in place".
> 

Reviewer: ```suggestion
> In Outlook on the Web, to be able to see Override in Policy Tip, the DLP policy needs to be turned on since it won't be displayed in Test Mode (with notifications). In DLP configuration, some restriction action should also be in place.
```
